docs/src/main/asciidoc/_migration-guide-for-4.0.adoc

```diff
@@ -301,49 +301,75 @@ crash.
 [cols="<30,<~", options="header"]
 |===
 |Removed dependencies |Description
+|com.fasterxml.jackson.core:jackson-databind |Please include the jackson databind dependency if you want to use Jackson Databind.
+|io.projectreactor.netty:reactor-netty |Please include the reactor netty dependency if you want to use Reactor Netty.
 |org.springframework.boot:spring-boot-starter-validation |Please include the validation starter if you want to use Hibernate Validator.
+|org.springframework.boot:spring-boot-starter-webflux |Please include the webflux starter if you want to use AAD condition access.
 |===
 
 [#configuration-spring-cloud-azure-starter-active-directory]
 ===== SDK Configuration Changes
 
 . All configuration property names changed the prefix from `azure.activedirectory` to `spring.cloud.azure.active-directory`.
-. New property `spring.cloud.azure.active-directory.enabled=true` is necessary to enable related features.
+. New property `spring.cloud.azure.active-directory.enabled=true` is required to enable related features.
```

Comment:
New property `spring.cloud.azure.active-directory.enabled=true` is added to allow enable / disable AAD related features. The default vaule is `true`.

Reply:
OK, the default value is false.